packages/orm/src/index.ts

```diff
@@ -421,6 +426,11 @@ export function getRelations(entity: Entity): AbstractRelationImpl<any>[] {
   return Object.values(entity).filter((v) => v instanceof AbstractRelationImpl);
 }
 
+export function getConstructorFromTaggedId(id: string): EntityConstructor<any> {
+  const [tag] = id.split(":");
```

Comment:
There is a `tagDelimiter` `const` in `keys.ts` that in theory "owns" the knowledge of `:`s, maybe put a method in `keys.ts` to get the tag from an id?